packages/polywrap-uri-resolvers/polywrap_uri_resolvers/aggregator/uri_resolver_aggregator.py

```diff
@@ -0,0 +1,19 @@
+from typing import List, Optional
+
+from polywrap_core import IUriResolver, Uri, IUriResolutionContext, Client
+from polywrap_result import Result, Ok
+
+from .uri_resolver_aggregator_base import UriResolverAggregatorBase
+
+class UriResolverAggregator(UriResolverAggregatorBase):
+    resolvers: List[IUriResolver]
+    name: Optional[str]
+
+    def __init__(self, resolvers: List[IUriResolver]):
+      self.resolvers = resolvers
+
+    def get_step_description(self) -> str:
```

Comment:
Shouldn't this be get_step_name?

Reply:
we are following the `UriResolverAggregatorBase` interface and I think it makes sense to keep it with description